File: kanban-board/kanban-board-app/src/main/java/com/niit/kanbanboardapp/domain/Task.java
package com.niit.kanbanboardapp.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.Date;
import java.util.List;


@Data
@AllArgsConstructor
@NoArgsConstructor
public class Task {
    private String taskId;
    private String taskName,taskDescription,priority;
    private Date assignedDateAndTime,updatedDateAndTime,deadlineDateAndTime;
    private String status,assignee;
    private List<String> assignedTo;
    private List<String> watcher;

}

File: kanban-board-ui/src/app/models/task.ts
export type Task = {
    taskId:string,
    taskName : string,
    taskDescription : string,
    priority : string,
    assignedDateAndTime : string  ,
    updatedDateAndTime :  string  ,
    deadlineDateAndTime :  string ,
    status : string,
    assignee : string,
    assignedTo: string[] ,
    watcher : string[] 
}